kube-controllers/pkg/controllers/node/hostendpoints.go

```diff
@@ -303,25 +303,39 @@ func (c *autoHostEndpointController) generateAutoHostendpointName(nodeName strin
 // that should set on the auto hostendpoint.
 func (c *autoHostEndpointController) getAutoHostendpointExpectedIPs(node *libapi.Node) []string {
 	expectedIPs := []string{}
+	IPMap := make(map[string]struct{}) // used to avoid adding duplicates to expectedIPs
```

Comment:
nit: this should be lowercase since it's a local variable.

e.g., `ipMap`

Reply:
Right, sorry about that. Will fix